README.md

```diff
@@ -1,31 +1,31 @@
-# unc0ver
-### The most advanced jailbreak tool
+# Unc0ver
+### The most advance jailbreak tool
 ![unc0ver logo](https://raw.githubusercontent.com/pwn20wndstuff/Undecimus/master/Undecimus/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60%403x.png?token=AlyO4xDujoguob2DCFfUbNI8jO82OyCgks5bx5ZPwA%3D%3D)
 
-unc0ver jailbreak for iOS 11.0 - 11.4b3<br/>
+unc0ver jailbreak for iOS 11.0 - 11.4.1<br/>
```

Comment:
Should update after the release of iOS 12 jailbreak to update.

Reply:
We’ll see that